example_simple_regression.py

```diff
@@ -0,0 +1,70 @@
+"""This file is used to create and compare two models on a particular dataset.
+It provides examples of reading from both csv and SQL Server. Note that this
+example can be run as-is after installing healthcare.ai. After you have
+found that one of the models works well on your data, move to Example2
+"""
+import pandas as pd
+import time
+
+import healthcareai.common.file_io_utilities as io
```

Comment:
I recommend we import this as something other than io - which is a core library name.
```
from healthcareai.common import file_io_utilities as hcio
```